main.nf

```diff
@@ -696,14 +813,16 @@ process multiqc {
     file multiqc_config
     file (fastqc:'fastqc/*') from fastqc_results.collect()
     file ('trimgalore/*') from trimgalore_results.collect()
-    file ('bwa/*') from bwa_logs.collect()
+    file ('bwa/*') from bwa_stdout.collect()
```

Comment:
Not sure if you'll get conflicting filenames here when running with multiple samples as we had in the RNA pipeline recently with NF v0.26?

If so, then you can get sequential filenames like so:
```groovy
file ('bwa/sample_*') from bwa_stdout.collect()
```

See https://www.nextflow.io/docs/latest/process.html#multiple-input-files

Reply:
No, we don't have any conflicting filenames here.